libs/langchain/langchain/chains/combine_documents/stuff.py

```diff
@@ -1,21 +1,54 @@
 """Chain that combines documents by stuffing into context."""
-
-from typing import Any, Dict, List, Optional, Tuple
+from typing import Any, Dict, List, Optional, Tuple, Union
 
 from langchain_core.documents import Document
+from langchain_core.language_models import LanguageModelInput
+from langchain_core.messages import BaseMessage
+from langchain_core.output_parsers import BaseOutputParser
 from langchain_core.prompts import BasePromptTemplate, format_document
 from langchain_core.prompts.prompt import PromptTemplate
 from langchain_core.pydantic_v1 import Extra, Field, root_validator
+from langchain_core.runnables import Runnable, RunnableParallel
 
 from langchain.callbacks.manager import Callbacks
-from langchain.chains.combine_documents.base import (
-    BaseCombineDocumentsChain,
-)
+from langchain.chains.combine_documents.base import BaseCombineDocumentsChain
 from langchain.chains.llm import LLMChain
 
+LanguageModelLike = Union[
+    Runnable[LanguageModelInput, str], Runnable[LanguageModelInput, BaseMessage]
+]
+
+
+def create_stuff_documents_chain(
+    llm: LanguageModelLike,
+    prompt: BasePromptTemplate,
+    output_parser: BaseOutputParser,
+    document_input_key: str,
```

Comment:
let's keep things simple and remove any args related to input or output keys. if users want to customize the keys they can just transform the dict before/after calling the func

Reply:
Agreed